pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -702,40 +705,78 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 	}
 
 	for _, v := range objects {
+		label := labels.SelectorFromSet(labels.Set(map[string]string{transformer.Selector: v.(meta.Object).GetName()}))
+		options := api.ListOptions{LabelSelector: label}
 		switch t := v.(type) {
 		case *extensions.Deployment:
 			//delete deployment
-			rpDeployment, err := kubectl.ReaperFor(extensions.Kind("Deployment"), client)
+			deployment, err := client.Deployments(namespace).List(options)
 			if err != nil {
 				return err
 			}
-			//FIXME: gracePeriod is nil
-			err = rpDeployment.Stop(namespace, t.Name, TIMEOUT*time.Second, nil)
-			if err != nil {
-				return err
+			komposeLabel := map[string]string{transformer.Selector: t.Name}
+			if len(deployment.Items) == 0 {
```

Comment:
@kadel If we remove this line, there will be no output of `kompose down` is there is nothing deployed and you mentioned earlier that we want at least want to give an error in this situation.